dss_s3benchmark/README.md

```diff
@@ -35,7 +37,7 @@ Otherwise, to build the s3-benchmark executable, you must issue this following c
 
 *	go get -u github.com/aws/aws-sdk-go/aws/...
 *	go get -u github.com/aws/aws-sdk-go/service/...
-*	go get -u github.com/pivotal-golang/bytefmt/...
+*	go get -u code.cloudfoundry.org/bytefmt
```

Comment:
What prompted this change ? was working earlier

Reply:
the above link was throwing exception to download from code.cloudfoundry.org/bytefmt. When I used the browser with the link https://github.com/cloudfoundry/bytefmt, the README file says
```Note: This repository should be imported as code.cloudfoundry.org/bytefmt.```